mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraMessageModule.java

```diff
@@ -47,7 +50,20 @@
     private final List<CassandraType> types;
 
     public CassandraMessageModule() {
-        tables = Collections.singletonList(
+        tables = ImmutableList.of(
+            new CassandraTable(CassandraMessageIdTable.TABLE_NAME,
+                SchemaBuilder.createTable(CassandraMessageIdTable.TABLE_NAME)
+                    .ifNotExists()
+                    .addPartitionKey(CassandraMessageIds.MAILBOX_ID, timeuuid())
+                    .addClusteringColumn(CassandraMessageIds.IMAP_UID, bigint())
+                    .addColumn(CassandraMessageIds.MESSAGE_ID, timeuuid())),
+            new CassandraTable(CassandraImapUidTable.TABLE_NAME,
+                SchemaBuilder.createTable(CassandraImapUidTable.TABLE_NAME)
+                    .ifNotExists()
+                    .addPartitionKey(CassandraMessageIds.MESSAGE_ID, timeuuid())
+                    .addClusteringColumn(CassandraMessageIds.MAILBOX_ID, timeuuid())
+                    .addColumn(CassandraMessageIds.IMAP_UID, bigint())),
```

Comment:
Here, it means if you want tu duplicate the same message, two time, in a mailbox, it will appear one ?

Could you add the following test in MessageMapperTest ?

```
 - Max take a message in INBOX and copies it two time in his prefered forlder
 - makes lists his messages of his prefered folder, and should see it two times
```

I believe you should use a addClusteringColumn too....